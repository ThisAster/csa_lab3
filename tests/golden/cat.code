[{"opcode": "ld", "arg_type": "immediate", "arg": 1},
 {"opcode": "je", "arg_type": "address", "arg": 7},
 {"opcode": "in", "arg_type": "address", "arg": 0},
 {"opcode": "st", "arg_type": "address", "arg": 0},
 {"opcode": "ld", "arg_type": "address", "arg": 0},
 {"opcode": "out", "arg_type": "address", "arg": 0},
 {"opcode": "jmp", "arg_type": "address", "arg": 0},
 {"opcode": "hlt", "arg_type": "none", "arg": 0}]