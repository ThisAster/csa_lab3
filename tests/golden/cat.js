var char
while (1) {
   char = input()
   print(char)
}
