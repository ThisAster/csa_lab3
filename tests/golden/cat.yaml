js: cat.js
code: cat.code
data: cat.data
txt: cat.txt
translator_log: cat-1.log
machine_log: cat-2.log
